### PR TITLE
Added payloadData

### PR DESCRIPTION
**Pull request by HripsimeMartirosyan1990**

### CODE REVIEW

File: Sources/ShortcutFoundation/Extensions/String+Extension.swift [hunk 1]
```diff
@@ -88,6 +88,8 @@ public extension String {
         case .weekdayDayMonthYear: return "EEEE, dd MMM, YYYY"
         case .weekdayDayMonthTime: return "EEEE, dd MMM, h:mm a"
         case .dayMonthTime: return "dd MMM, h:mm a"
+        case .dayMonth: return "dd/MM"
```
Comment (lemonandlime) [inline Sources/ShortcutFoundation/Extensions/String+Extension.swift:91]: This does not feel like standarlized date formats.

File: Sources/ShortcutFoundation/Core/Network/Core/Calls/NetworkingClient+NetworkingJSONDecodable.swift [hunk 2]
```diff
@@ -13,7 +13,7 @@ public typealias NetworkPublisher<Response> = AnyPublisher<Response, NetworkingE
 
 public extension NetworkingClient {
 
-    func get<ResponsePayload: Decodable, Payload: Encodable>(_ route: String, params: Payload) -> NetworkPublisher<ResponsePayload> {
+    func get<ResponsePayload: Decodable, Payload: Encodable, errorPeyload: Encodable>(_ route: String, params: Payload, error: errorPeyload) -> NetworkPublisher<ResponsePayload> {
```
Comment (lemonandlime) [inline Sources/ShortcutFoundation/Core/Network/Core/Calls/NetworkingClient+NetworkingJSONDecodable.swift:16]: typo: `errorPeyload`

Comment (lemonandlime) [reply Sources/ShortcutFoundation/Core/Network/Core/Calls/NetworkingClient+NetworkingJSONDecodable.swift:16]: Also I do not want to send this in to all my requests. so you will have to create an overload.

**Review by lemonandlime — CHANGES_REQUESTED** [review]

**Comment by lemonandlime** [comment]
Since this is not something that happens all the time that you need to decode error payload. Maybe we should just add it to the error as `Data` if it exists. Then we would not need to send in all that stuff in the init method of the get function. TYou could just check the throwing error if the network call fails and if you get data then manually decode to whatever type you expect to get in that case

**Comment by lemonandlime** [comment]
@HripsimeMartirosyan1990 What do you say can we close this or do you wanna fix the issues the PR is getting a bit old.

**Comment by HripsimeMartirosyan1990** [comment]
> @HripsimeMartirosyan1990 What do you say can we close this or do you wanna fix the issues the PR is getting a bit old.

@lemonandlime those changes is used in Pangeo maybe we still need them to be merged

**Pull request closed without merging**